javascript-arrays/main.js

```diff
@@ -0,0 +1,21 @@
+var colors = ['red', 'white', 'blue'];
+console.log('value of colors[0]:', colors[0]);
+console.log('value of colors[1]:', colors[1]);
+console.log('value of colors[2]:', colors[2]);
+
+console.log('America is ' + colors[0] + ', ' + colors[1] + ', and ' + colors[2] + '.');
+
+colors[2] = 'green';
+console.log('Mexico is ' + colors[0] + ', ' + colors[1] + ', and ' + colors[2] + '.');
+
+console.log(colors);
```

Comment:
just a tip, it's always a good idea to label your console logs: `console.log('value of colors:', colors);`

Reply:
Got it! I accidentally forgot, but I'll remember it now.